app/controllers/api/v1/instances_controller.rb

```diff
@@ -3,8 +3,10 @@
 class Api::V1::InstancesController < Api::BaseController
   skip_before_action :require_authenticated_user!, unless: :whitelist_mode?
 
+  vary_by ''
+
   def show
-    expires_in 3.minutes, public: true
+    cache_even_if_authenticated!
```

Comment:
I think it should still vary by `Authorization`, as this embeds an account (the contact account), which has user-dependent relationships and such.

Reply:
The relationships aren't part of the object though

Reply:
You're right, my bad!